search/common.hpp

```diff
@@ -17,6 +19,6 @@ using Locales =
     base::SafeSmallSet<static_cast<uint64_t>(CategoriesHolder::kMaxSupportedLocaleIndex) + 1>;
 
 /// Upper bound for max count of tokens for indexing and scoring.
-int constexpr MAX_TOKENS = 32;
-int constexpr MAX_SUGGESTS_COUNT = 5;
+size_t constexpr kMaxNumTokens = 32;
```

Comment:
Can you guarantee ODR-usage for these constants?

Reply:
They are currenly odr-used as far as I can see.
Do you think we should define them?

Reply:
Up to you. While it links successfully, no need to define them.